templates/zoo.cfg.j2

```diff
@@ -5,11 +5,7 @@ clientPort={{ client_port }}
 initLimit={{ init_limit }}
 syncLimit={{ sync_limit }}
 
-{#
-Also consider:
-server.{{server.id}}={{server.host}}:2888:3888
-#}
 
 {% for server in zookeeper_hosts %}
-server.{{loop.index}}={{server.host}}:2888:3888
+server.{{server.id}}={{server.host}}:2888:3888
```

Comment:
> gives an operator full control over the identity of the nodes

you got the point here, although I wonder if both dict and array can be supported